### PR TITLE
[QUIZ-111] [QUIZ-112] Fix toggles

### PR DESCRIPTION
**Pull request by heathermusson**
Toggles are not currently question specific -> clicking show question on one toggle shows the question text for all questions. This PR fixes that.

screencast 2021-07-02 15-47-27

### CODE REVIEW

File: src/instructor/QuestionDetails.tsx [hunk 1]
```diff
@@ -19,29 +19,26 @@ interface Props {
...
-    showQuestion,
-    setShowQuestion,
-    showLabels,
-    setShowLabels,
-    onSendQuestion,
-    onRemoveQuestion,
-  } = props;
-  const { question, type, answers } = props.question;
+export const QuestionDetails = ({
+  question: questionFromProps,
```
Comment (heathermusson) [inline src/instructor/QuestionDetails.tsx:28]: After @BenLanglois PR gets merged (#44) I can remove the prop renaming.

**Review by BenLanglois — APPROVED** [review]

File: src/instructor/QuestionDetails.tsx [hunk 2]
```diff
@@ -51,39 +48,37 @@ export const QuestionDetails = (props: Props) => {
...
-      )}
       {isQuestionLive ? (
-        <>
-          <QuestionGraphDetails
-            question={props.question}
-            isLive={isLive}
-            setIsQuestionLive={setIsQuestionLive}
-          />
-        </>
+        <QuestionGraphDetails
```
Comment (alexgirard) [inline src/instructor/QuestionDetails.tsx:60]: ty for removing the fragment!

File: src/instructor/QuestionDetails.tsx [hunk 3]
```diff
@@ -51,39 +48,37 @@ export const QuestionDetails = (props: Props) => {
...
+          question={questionFromProps}
+          isLive={isLive}
+          setIsQuestionLive={setIsQuestionLive}
+        />
       ) : (
-        <StretchColumn gridGap={4}>
+        <>
```
Comment (alexgirard) [inline src/instructor/QuestionDetails.tsx:66]: why was the stretch column removed? it had padding?

**Review by alexgirard — APPROVED** [review]

File: src/instructor/QuestionDetails.tsx (hunk 3 above)
Comment (heathermusson) [reply src/instructor/QuestionDetails.tsx:66]: I'm now just using the one on the outside of the ternary operator. From what I could tell there is no visual difference. But I can double check.

File: src/instructor/QuestionDetails.tsx (hunk 1 above)
Comment (BenLanglois) [reply src/instructor/QuestionDetails.tsx:28]: #44 has been merged!

**Pull request merged**